test/exercises/leap_test.rb

```diff
@@ -0,0 +1,59 @@
+require "test_helper"
+
+class LeapTest < Minitest::Test
+  def test_date_leap_method_is_disapproved
+    source = "
+      class Year
+        def self.leap?(year)
+          Date.leap?(year)
+        end
+      end
+    "
+
+    results = Leap::Analyze.(source)
+    assert_equal :disapprove, results[:status]
+    assert_equal ["ruby.leap.std_lib"], results[:comments]
+  end
+
+  def test_date_julian_leap_method_is_disapproved
+    source = "
+      class Year
+        def self.leap?(year)
+          Date.julian_leap?(year)
+        end
+      end
+    "
```

Comment:
Consider using Heredocs for strings like this.  Something like

```suggestion
    source = <<~SOURCE
      class Year
        def self.leap?(year)
          Date.julian_leap?(year)
        end
      end
    SOURCE
```

Reply:
This would be inconsistent with the other tests, so I think we should change all of them, or leave all of them as they are.

Reply:
Yep, I did not want to have to provide individual examples, hoping you would pick up on the pattern!

Reply:
Please use Heredocs, the Heredoc provided will trim to the indentation, and will end up being "smaller" in effective size, since it will trim additional unwanted space on the left hand side.

Reply:
I meant that it would be inconsistent with the tests in other files. I will use Heredoc, but we should create an issue to update the existing tests.

Reply:
Ah, there may be a reason that it does, perhaps it is important that the strings are not trimmed.  Test and find out?  It seems odd to not use Heredocs.

I was just getting ready to look at the other files to see if they were using Heredocs as well, as I knew it would be reasonable to see this if it was there.

But we can worry about transitioning to the Heredocs for the other files, when we are done with this one (and before starting with the next one you want to do).

Reply:
I used Heredoc, and it works.